server_admin/topics/clients/client-saml.adoc

```diff
@@ -54,6 +54,11 @@ Include AuthnStatement::
   This is enabled by default, which means that `AuthStatement` element will be included in login responses. Note that setting this to off
   would prevent the client from determining the maximum session length which could result into never expiring client session.
 
+Force Artifact Binding::
+  When turned on, this will force {project_name} to send artifact messages instead of SAML messages via POST and redirect,
+  even when the client hasn't asked for the binding during login. This must be set if artifact binding is to used with
+  idp-initiated login. To use artifact messages also during logout, it is necessary to also configure `Logout Service Redirect Binding URL`.
+
```

Comment:
Please make this consistent (format-wise) with the rest of the book, i.e. `_POST_` and `_Redirect_`, and also `_Artifact_`.
Also use full form, i.e. _has not_ rather than _hasn't_